front-end/src/components/Dashboard/index.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import styled from 'styled-components';
+import SideBarNav from "../../shared/SideBarNav";
+import jwt_decode from 'jwt-decode'
+
+
+
+
+class Dashboard extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            created_at: '',
+            id: '',
+            email: '',
+            password: '',
+            isLoaded: false
+        };
+    }
+        componentDidMount() {
+            let token =  localStorage.getItem('access_token');
+            let decoded = jwt_decode(token);
+            console.log({decoded});
+            return fetch("http://localhost:3000/users/" + decoded.id, {
+                method: 'GET',
+                headers:{
+                    'Accept': 'application/json',
+                    'Content-Type': 'application/json',
+                    'Authorization': 'Bearer '  + token ,
+                }
+
+            })
+                .then(res => res.json())
+                .then(
+                    (result) => {
+                        console.log(result[0]);
+                        this.setState({
+                            isLoaded: true,
+                            id: result[0].id,
+                            email: result[0].email,
+                            password: result[0].password,
+                            created_at: result[0].created_at
+                        });
+
+                    },
+                    (error) => {
+                        this.setState({
+                            isLoaded: true,
+                            error
+                        });
+                    }
+                )
+
+        }
+
+
+
+    render() {
+        return (
+            <DashboardContainer>
+
+                <SideBarNav>
+                </SideBarNav>
+                <RightPanel>
+                    <DashboardContentContainer>
+                        <DashboardContentHeading>
+                            Dashboard Content Header
+                        </DashboardContentHeading>
+                        <DashboardContent>
+                            <div>Dashboard Content</div>
+                            <div>UserID: {this.state.id}</div>
+                            <div>Email: {this.state.email}</div>
+                            <div>Password: {this.state.password} </div>
+                            <div>Created: {this.state.created_at}</div>
```

Comment:
I don't think the user would need to see when their account was specifically created, and I think it'd be dangerous to have their password in plain view.

Reply:
Removed